pkgs/by-name/ni/nixos-render-docs/src/nixos_render_docs/manual.py

```diff
@@ -542,8 +542,8 @@ def _parse(self, src: str, *, auto_id_prefix: None | str = None) -> list[Token]:
                 raise RuntimeError(f"redirection target {into} in line {token.map[0] + 1} is empty!")
             # we use blender-style //path to denote paths relative to the origin file
             # (usually index.html). this makes everything a lot easier and clearer.
-            if not into.startswith("//") or '/' in into[2:]:
-                raise RuntimeError("html:into-file must be a relative-to-origin //filename", into)
+            # if not into.startswith("//") or '/' in into[2:]:
+            #     raise RuntimeError("html:into-file must be a relative-to-origin //filename", into)
```

Comment:
Given the errors me and @fricklerhandwerk saw on the built output, I think removing these lines might need some more thought. Which is why I hope @pennae can show some light on this.

Reply:
For context as to why these two lines are relevant, is that we're adding in output paths using the [`html:into-file`](https://github.com/NixOS/nixpkgs/blob/63839c6b9a84a85ffc167c4f9c9966510f1d00b3/nixos/doc/manual/manual.md?plain=1#L50-L52) syntax. But this only works as long as the file path does not have `/` in it; meaning no sub-paths.